### PR TITLE
AG-9066 Fix test names

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-9066

* All test files now have one describe block
* Changes `for (...) { test('...') }` to `it.each` etc. so we get errors if there are no test cases (we had one example of this)

Note this regenerates image snapshots. If it looks like an image changed, it's because it never met the threshold before to be considered 'changed'

### CODE REVIEW

File: packages/ag-charts-community/src/chart/galleryExamplesUSTimezone.test.ts [hunk 1]
```diff
@@ -50,15 +50,21 @@ describe('Gallery Examples (US TZ)', () => {
     });
 
     const ctx = setupMockCanvas();
-    for (const [exampleName, example] of Object.entries(TIME_AXIS_EXAMPLES)) {
-        it(`for ${exampleName} it should create chart instance as expected`, async () => {
+    // Missing time axis examples
+    it.skip.each(Object.entries(TIME_AXIS_EXAMPLES))(
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/galleryExamplesUSTimezone.test.ts:54]: why skipped?

File: packages/ag-charts-community/src/chart/mapping/themes.test.ts [hunk 2]
```diff
@@ -17,7 +16,7 @@ import { themes } from './themes';
 expect.extend({ toMatchImageSnapshot });
 
 /* eslint-disable no-console */
-describe('ThemesValidation', () => {
+describe('validateChartThemeObject', () => {
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/mapping/themes.test.ts]: Should this be called `'Themes'` to match the other changes matching their file names?

**Review by lsjroberts — APPROVED** [review]
LGTM apart from minor questions

File: packages/ag-charts-community/src/chart/galleryExamplesUSTimezone.test.ts (hunk 1 above)
Comment (jacobp100) [reply packages/ag-charts-community/src/chart/galleryExamplesUSTimezone.test.ts:54]: There weren't any examples. It was missed before in the for loop, but now jest throws an error

File: packages/ag-charts-community/src/chart/mapping/themes.test.ts (hunk 2 above)
Comment (jacobp100) [reply packages/ag-charts-community/src/chart/mapping/themes.test.ts]: Updated

**Pull request merged**